routes/admin.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express');// pass passport for configuration
+const router = express.Router();
+const { isLoggedIn, isAdmin } = require('./utils/middleware') ;
+const user = require('./model/usermodel');
+
+
+
+// router.delete('/admin/users',(req,res,next)=>{
+//     user.remove({},(err,user)=>{
+//         res.json(user);
+//     })
+// });
+
+router.get("/admin/addCompany" , (req,res) => {
```

Comment:
These routes must be post and delete routes.
Also, use isLoggedIn and isAdmin middleware on these routes so that they can be accessed by admins only.